python/cugraph/cugraph/community/subgraph_extraction.py

```diff
@@ -56,33 +57,12 @@ def subgraph(G, vertices):
 
     """
 
-    G, isNx = ensure_cugraph_obj_for_nx(G)
-    directed = G.is_directed()
+    warning_msg = (
+        "This call is deprecated and will be refactored "
+        "in the next release. Please call 'cugraph.induced_subgraph()' instead"
+    )
+    warnings.warn(warning_msg, PendingDeprecationWarning)
```

Comment:
Should use `DeprecationWarning` since it's now being officially deprecated.
```suggestion
    warnings.warn(warning_msg, DeprecationWarning)
```